From 8b12b7e21cadf9a43782a32c244f9d16d14e4cb3 Mon Sep 17 00:00:00 2001
From: onenowy <92918392+onenowy@users.noreply.github.com>
Date: Sun, 19 Nov 2023 14:51:47 +0900
Subject: [PATCH] hwdb: add Predator PHN16-71

enable microphone mute, predator sense button and fix the keyboard backlight up button changes display brightness.

(cherry picked from commit a3f5976ded023257f6299ca07b9749fd1483c0d2)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index e981ff4e14..acb23bfbf7 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -225,6 +225,12 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnPackard*Bell*:pn*:*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnSwiftSF314-511:pvr*
  KEYBOARD_KEY_8a=f20					# Fn+F12, microphone mute
 
+# Predator PHN16-71
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnPredatorPHN16-71:*
+ KEYBOARD_KEY_ef=unknown                                # Keyboard backlight up
+ KEYBOARD_KEY_f5=prog1                                  # "predator sense" button
+ KEYBOARD_KEY_66=micmute                                # Microphone mute button
+
 ###########################################################
 # Alienware
 ###########################################################
